Experimental/MultiplayerUseCases/Assets/UseCaseSamples/NetvarVsRpc/Scripts/ColorManager.cs

```diff
@@ -1,7 +1,7 @@
 using Unity.Netcode.Samples.MultiplayerUseCases.Common;
 using UnityEngine;
 
-namespace Unity.Netcode.Samples.MultiplayerUseCases.NetVarVsRPC
+namespace Unity.Netcode.Samples.MultiplayerUseCases.NetVarVsRpc
```

Comment:
Nitpick: RPC is an acronym is there any reason why this is being changed?

Reply:
We have a new standard regarding naming to have "Rpc" instead of "RPC". Unfortunately I do not know anymore where the standard is coming from.